ui/widgets/top-bar/src/components/topbar-component.tsx

```diff
@@ -108,7 +108,11 @@ const TopbarComponent = (props: RootComponentProps) => {
 
   React.useEffect(() => {
     const isLoadingProfile = profileDataReq.isLoading !== undefined && profileDataReq.isLoading;
-    if (loginQuery.data?.ethAddress && !isLoadingProfile) {
+    if (
+      loginQuery.data?.ethAddress &&
+      !isLoadingProfile &&
+      singleSpa.getMountedApps().find(app => !app.includes('app-auth'))
```

Comment:
@kenshyx @SeverS Had to use this to prevent the update modal from opening automatically on step 5 of sign-up. Is this acceptable, any better approaches?